### ISSUE TITLE
Compile error ubuntu 22.04 please recompile with fpie

### ISSUE THREAD

**Issue by dcutugno** [xmrig/xmrig#3423]
When I try to compile I get this error at the end:

100%] Linking CXX executable xmrig
/usr/bin/ld: CMakeFiles/xmrig.dir/src/base/tools/cryptonote/crypto-ops.c.o: relocation R_X86_64_32S against symbol `ge_base' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: failed to set dynamic section sizes: bad value
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/xmrig.dir/build.make:3856: xmrig] Error 1
make[1]: *** [CMakeFiles/Makefile2:182: CMakeFiles/xmrig.dir/all] Error 2
make: *** [Makefile:91: all] Error 2

**Comment by SChernykh** [comment]
How exactly did you compile it? Did you follow instructions from https://xmrig.com/docs/miner/build/ubuntu ? If yes, was it basic or advanced build instructions?

**Comment by dcutugno** [comment]
Yes followed both basic and advanced Il giorno 17 feb 2024, alle ore 20:27, SChernykh ***@***.***> ha scritto:﻿
How exactly did you compile it? Did you follow instructions from https://xmrig.com/docs/miner/build/ubuntu ? If yes, was it basic or advanced build instructions?

**Comment by dcutugno** [comment]
An update to this issue, the error was due to have installed from source gcc 13.0.1 experimental, now reverting back to gcc 11 it compiled fine with basic and advanced, I will try switching to most recent gcc with update-alternatives and recompile it...
